swagger/swagger.yml

```diff
@@ -47,7 +47,7 @@ paths:
           in: query
           type: integer
           format: int32
```

Comment:
if you remove `format: int32`
then you don't need to type-cast in code.

Reply:
Done ! Please check